src/lib/util/random-id.ts

```diff
@@ -1,5 +1,5 @@
-import { v4 as uuidv4 } from 'uuid';
+import { randomUUID } from 'node:crypto';
 
 export const randomId = (): string => {
```

Comment:
Be cool if we could decide on whether to use this everywhere or nowhere. But maybe a different PR

Reply:
We use this function in 201 places but we just use randomUUID directly in some helper methods, so I think we can assume we're using randomId currently